index.js

```diff
@@ -79,7 +79,7 @@ var Sqlite3Driver = Base.extend({
       constraint.push('UNIQUE');
     }
 
-    if (spec.defaultValue) {
+    if (spec.defaultValue !== undefined) {
```

Comment:
What do you exactly want  to do?

In this case there are also null values accepted. Please specify more what you want to do.

What I currently understand from your title:

You want to support boolean values such as true and false or 0 and 1. If this is the case, maybe it is more suitable to check if `spec.defaultValue` is equal to false or 0.

For example:

```javascript
if (spec.defaultValue || spec.defaultValue === false || spec.defaultValue === 0)
```